packages/stores/types.ts

```diff
@@ -269,45 +253,28 @@ type TNotificationStore = {
 
 type TTradersHubStore = {
     closeModal: () => void;
-    content_flag: 'low_risk_cr_eu' | 'low_risk_cr_non_eu' | 'high_risk_cr' | 'cr_demo' | 'eu_demo' | 'eu_real' | '';
-    combined_cfd_mt5_accounts: DetailsOfEachMT5Loginid &
-        {
-            short_code_and_region: string;
-            login: string;
-            sub_title: string;
-            icon: 'Derived' | 'Financial' | 'Options' | 'CFDs';
-        }[];
-    openModal: (modal_id: string, props?: unknown) => void;
-    selected_account: {
-        login: string;
-        account_id: string;
-    };
+    content_flag: any;
     is_low_risk_cr_eu_real: boolean;
+    openModal: (modal_id: string, props?: any) => void;
     is_eu_user: boolean;
-    setTogglePlatformType: (platform_type: string) => void;
     is_real: boolean;
+    selected_account_type: string;
     selectRegion: (region: string) => void;
+    CFDs_restricted_countries: boolean;
     financial_restricted_countries: boolean;
-    selected_account_type: string;
-    no_CR_account: boolean;
-    no_MF_account: boolean;
+    gaming_restricted_countries: boolean;
+    is_cfd: boolean;
+    is_financial: boolean;
+    is_gaming: boolean;
+    is_high_risk_cr_eu_real: boolean;
 };
 
-/**
- * This is the type that contains all the `core` package stores
- */
-export type TCoreStores = {
+export type TRootStore = {
```

Comment:
why are you renaming TCoreStores to TRootStore
It will break the other code and Circle CI